.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 npm-debug.log
 .DS_Store
+.idea
```

Comment:
I don't believe `.idea` belongs here.  Instead, you should have it in your [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).